src/main/java/org/example/Main.java

```diff
@@ -1,22 +1,31 @@
 package org.example;
 import java.util.Scanner;
 
-import static java.lang.System.in;
-import static java.lang.System.out;
+import static java.lang.System.*;
 
 public class Main {
 
     public static final String CORRECT_NUMBER = "5";
+    public static  int MAX_TRIES = 5;
+    public static int NUMBER_TRIES= 0;
 
     public static void main(String[] args) {
-        Scanner number = new Scanner(in);
-        out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
-        String name = number.nextLine();
-        out.println(name);
-        if (CORRECT_NUMBER.equals(name)) {
-            out.println("Wszedles");
-        } else {
-            out.println("Nie prawidlowe haslo");
+        while (NUMBER_TRIES < MAX_TRIES) {
+            Scanner number = new Scanner(in);
+            out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
```

Comment:
you can use String.format() or printf() method here to make it more readable. 
https://www.baeldung.com/string/format
https://www.baeldung.com/java-printstream-printf